lib/owl_plot.ml

```diff
@@ -1593,14 +1593,24 @@ let scatterhist = None
 
 (* other plots *)
 
+let _rotate270_ x =
+  let w, h = Owl_dense_matrix.D.shape x in
```

Comment:
Thanks a lot. The `transpose` + `slice` can do exactly the rotate magic required:

```ocaml 
let _rotate270_ x =
  x |> Mat.transpose |> Mat.slice [[]; [-1;0]] 
```
But of course a `rotate` function would be great to have :)